R/datadump-valider.R

```diff
@@ -115,9 +115,11 @@ lag_regelsett = function(kodebok, oblig = TRUE, rekkefolge = TRUE) {
       pmap(function(varnamn, gverdi) {
         rlang::new_function(
           alist(df = ),
-          rlang::expr(transmute_at(df,
-            .vars = vars(foo = !!varnamn),
-            .funs = ruler::rules(min_ok = (. >= !!gverdi) | is.na(.))
+          rlang::expr(mutate(df,
+            across(c(foo = !!varnamn),
```

Comment:
Her bør det vel stå `all_of(varnamn)` i staden for `c(foo = !!varnamn)`? Gjeld fleire tilsvarande plassar i fila.